rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json

```diff
@@ -654,38 +653,220 @@
+            },
+            {
+              "label": "Blueprint process",
+              "value": "rdapm:1002"
+            },
+            {
+              "label": "Collotype",
+              "value": "rdapm:1003"
+            },
+            {
+              "label": "Daguerreotype process",
+              "value": "rdapm:1004"
+            },
+            {
+              "label": "Engraving",
+              "value": "rdapm:1005"
+            },
+            {
+              "label": "Etching",
+              "value": "rdapm:1006"
+            },
+            {
+              "label": "Lithography",
+              "value": "rdapm:1007"
+            },
+            {
+              "label": "Photocopying",
+              "value": "rdapm:1008"
+            },
+            {
+              "label": "Photoengraving",
+              "value": "rdapm:1009"
+            },
+            {
+              "label": "Printing",
+              "value": "rdapm:1010"
+            },
+            {
+              "label": "White print process",
+              "value": "rdapm:1011"
+            },
+            {
+              "label": "Woodcut making",
+              "value": "rdapm:1012"
+            },
+            {
+              "label": "Photogravure process",
+              "value": "rdapm:1014"
+            },
+            {
+              "label": "Burning",
+              "value": "rdapm:1015"
+            },
+            {
+              "label": "Inscribing",
+              "value": "rdapm:1016"
+            },
+            {
+              "label": "Stamping",
+              "value": "rdapm:1017"
+            },
+            {
+              "label": "Embossing",
+              "value": "rdapm:1018"
+            },
+            {
+              "label": "Solid dot",
+              "value": "rdapm:1019"
+            },
+            {
+              "label": "Swell paper",
+              "value": "rdapm:1020"
+            },
+            {
+              "label": "Thermoform",
+              "value": "rdapm:1021"
+            }
+          ]
+        }
+      }
+    },
+    "bookFormat": {
+      "title": "Bibliographic format",
+      "description": "A proportional relationship between a whole sheet in a printed or manuscript resource, and the individual leaves that result if that sheet is left full, cut, or folded.",
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "type": "string",
+        "enum": [
+          "in-plano",
+          "2\u00ba",
+          "4\u00ba",
+          "8\u00ba",
+          "12\u00ba",
+          "16\u00ba",
+          "18\u00ba",
+          "24\u00ba",
+          "32\u00ba",
+          "36\u00ba",
+          "48\u00ba",
+          "64\u00ba",
+          "72\u00ba",
+          "96\u00ba",
+          "128\u00ba"
+        ],
+        "minLength": 2,
+        "form": {
+          "placeholder": "RDA recommended terms: in-plano, 2\u00ba, 4\u00ba, 8\u00ba, 12\u00ba, 16\u00ba, ..."
```

Comment:
placeholder does not make sense with enum. You need to provide form label, value list

Reply:
@reropag @jma But is it actually an enum, could the cataloger type in another value? There's no validation constrain in the specification file. ping @pronguen